src/app/homepage/pages/recipes/terminus/terminus.component.html

```diff
@@ -0,0 +1,95 @@
+  </p>
+  <span class="filename">app.module.ts</span>
+  <pre><code class="language-typescript">{{ appModule }}</code></pre>
+  <blockquote class="info">
+    <strong>Hint</strong>
+    If done correctly Nest will expose the defined health check(s), which are reachable through a GET request to the
+    defined route. For example <code>curl -X GET 'http://localhost:3000/health'</code>
+  </blockquote>
+  <h4>Creating a custom Health Indicator</h4>
+  <p>
+    In some cases the predefined health indicators provided by <code>@nestjs/terminus</code> do not cover all of your
+    health check requirements. In this case you can set up a custom health indicator according to your needs.
+  </p>
+  <p>
+    Lets get started by creating a service which will represent our custom health indicator. To get a basic
+    understanding
+    how a health indicator is structured, we will create an example <code>DogHealthIndicator</code>. This health
+    indicator should have the state "up", if every <code>Dog</code> object has the type <code>goodboy</code>,
+    otherwise it will throw an error, which then the health indicator will be seen as "down".
+  </p>
+  <blockquote class="error">
+    <strong>Hint</strong>
+    In a real world application it is seen as best practice to implement the logic
+    of fetching the required data in a separate service. The static array of dogs inside
+    the health indicator is just for demonstration purposes.
+  </blockquote>
+  <span class="filename">dog.health.ts</span>
+  <pre><code class="language-typescript">{{ dogHealthIndicator }}</code></pre>
+
+  <p>
+    The next thing we need to do is registering the health indicator as a provider.
+  </p>
+  <span class="filename">app.module.ts</span>
+  <pre><code class="language-typescript">{{ appModuleDog }}</code></pre>
+  <blockquote class="error">
+    <strong>Hint</strong>
+    In a real world application the <code>DogHealthIndicator</code> should be provided in
+    a separate Module, for example <code>DogModule</code>, which then get imported by the
+    <code>ApplicationModule</code>. But keep in mind to add the <code>DogHealthIndicator</code> to the
+    <code>exports</code> array of the <code>DogModule</code> and add the <code>DogModule</code> in
+    <code>imports</code> array of the <code>TerminusModule.forRootAsync()</code> parameter object.
+  </blockquote>
+  <p>
+    The last required thing to do is to add the now available health indicator in
+    the required health check endpoint. For that we go back to our <code>TerminusOptionsService</code>
+    and implement it to the <code>/health</code> endpoint.
+  </p>
+  <span class="filename">app.module.ts</span>
+  <pre><code class="language-typescript">{{ terminusOptionsServiceDog }}</code></pre>
+  <p>
+    If everything done correctly, the response of the server when request the <code>/health</code>
```

Comment:
Reword the sentence. Especially "the response of the server when request the [...]" is confusing to me as first time reader